incident/utils/charts.py

```diff
@@ -75,3 +75,10 @@ class IncidentBranches(TextChoices):
     CATEGORIES = 'CATEGORIES', 'Categories'
     ASSAILANT = 'ASSAILANT', 'Assailant'
     STATUS_OF_CHARGES = 'STATUS_OF_CHARGES', 'Status of Charges'
+
+
+def IncidentBranchesAndTags():
```

Comment:
This is probably not the best way to do it, but it was how I was able to get it working initially. Open to suggestions!